extensions/amp-analytics/0.1/vendors.js

```diff
@@ -742,54 +746,58 @@ export const ANALYTICS_CONFIG = /** @type {!JsonObject} */ ({
       'basePrefix': 'v=1&' +
-      '_v=a1&' +
-      'ds=${dataSource}&' +
-      '${anonymizeIP}&' +
-      '_s=${requestCount}&' +
-      'dt=${title}&' +
-      'sr=${screenWidth}x${screenHeight}&' +
-      '_utmht=${timestamp}&' +
-      'cid=${clientId}&' +
-      'tid=${account}&' +
-      'dl=${documentLocation}&' +
-      'dr=${externalReferrer}&' +
-      'sd=${screenColorDepth}&' +
-      'ul=${browserLanguage}&' +
-      'de=${documentCharset}',
+          '_v=a1&' +
+          'ds=${dataSource}&' +
+          '${anonymizeIP}&' +
+          '_s=${requestCount}&' +
+          'dt=${title}&' +
+          'sr=${screenWidth}x${screenHeight}&' +
+          '_utmht=${timestamp}&' +
+          'cid=${clientId}&' +
+          'tid=${account}&' +
+          'dl=${documentLocation}&' +
+          'dr=${externalReferrer}&' +
+          'sd=${screenColorDepth}&' +
+          'ul=${browserLanguage}&' +
+          'de=${documentCharset}',
       'baseSuffix': '&a=${pageViewId}&' +
-      'z=${random}',
+          'z=${random}',
       'pageview': '${host}/r/collect?${basePrefix}&' +
-      't=pageview&' +
-      'jid=${random}&' +
-      '_r=1' +
-      '${baseSuffix}',
+          't=pageview&' +
+          'jid=${random}&' +
+          '_r=1' +
+          '${baseSuffix}',
       'event': '${host}/collect?${basePrefix}&' +
-      't=event&' +
-      'jid=&' +
-      'ec=${eventCategory}&' +
-      'ea=${eventAction}&' +
-      'el=${eventLabel}&' +
-      'ev=${eventValue}' +
-      '${baseSuffix}',
+          't=event&' +
+          'jid=&' +
+          'ec=${eventCategory}&' +
+          'ea=${eventAction}&' +
+          'el=${eventLabel}&' +
+          'ev=${eventValue}' +
+          '${baseSuffix}',
       'social': '${host}/collect?${basePrefix}&' +
-      't=social&' +
-      'jid=&' +
-      'sa=${socialAction}&' +
-      'sn=${socialNetwork}&' +
-      'st=${socialTarget}' +
-      '${baseSuffix}',
+          't=social&' +
+          'jid=&' +
+          'sa=${socialAction}&' +
+          'sn=${socialNetwork}&' +
+          'st=${socialTarget}' +
+          '${baseSuffix}',
       'timing': '${host}/collect?${basePrefix}&' +
-      't=${timingRequestType}&' +
-      'jid=&' +
-      'plt=${pageLoadTime}&' +
-      'dns=${domainLookupTime}&' +
-      'tcp=${tcpConnectTime}&' +
-      'rrt=${redirectTime}&' +
-      'srt=${serverResponseTime}&' +
-      'pdt=${pageDownloadTime}&' +
-      'clt=${contentLoadTime}&' +
-      'dit=${domInteractiveTime}' +
-      '${baseSuffix}',
+          't=${timingRequestType}&' +
+          'jid=&' +
+          'plt=${pageLoadTime}&' +
+          'dns=${domainLookupTime}&' +
+          'tcp=${tcpConnectTime}&' +
+          'rrt=${redirectTime}&' +
+          'srt=${serverResponseTime}&' +
+          'pdt=${pageDownloadTime}&' +
+          'clt=${contentLoadTime}&' +
+          'dit=${domInteractiveTime}' +
+          '${baseSuffix}',
+      'user-error': '${host}/collect?${basePrefix}&' +
```

Comment:
pls follow the camel case naming convention. this might require changes in code, so you can do it in a separate PR. we can get this in first.